lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -220,6 +227,7 @@ def execute_statement(statement, parameters)
       success = true
     rescue Sequel::DatabaseConnectionError, Sequel::DatabaseError => e
       @logger.warn("Exception when executing JDBC query", :exception => e)
+      @metric_errors.increment(:jdbc_query_errors)
```

Comment:
I think it would be better if we remove the mention of jdbc and maybe have
`DatabaseConnectionError` and `DatabaseError` in two different metric?